integration_tests/mockApis/auth.ts

```diff
@@ -120,37 +122,15 @@ const token = () =>
     },
   })
 
-const stubUser = (name: string) =>
-  stubFor({
-    request: {
-      method: 'GET',
-      urlPattern: '/auth/api/user/me',
-    },
-    response: {
-      status: 200,
-      headers: {
-        'Content-Type': 'application/json;charset=UTF-8',
-      },
-      jsonBody: {
-        staffId: 231232,
-        username: 'USER1',
-        active: true,
-        name,
-        ...defaultProfile,
-      },
-    },
-  })
-
 export const defaultUserId = '70596333-63d4-4fb2-8acc-9ca55563d878'
 
-const defaultProfile = {
-  id: defaultUserId,
-  roles: [],
-  isActive: true,
-  apArea: apAreaFactory.build(),
-}
-
-const stubProfile = (roles = [], userId = defaultUserId, isActive = true, apArea = undefined) =>
+const stubProfile = (
```

Comment:
I wonder if it's worth just passing in a `User` object here? rather than the properties individually?